### PR TITLE
SF-1229: Add a changelog

### PR DESCRIPTION
**Pull request by Vic-Dev**

### CODE REVIEW

File: packages/@storefront/core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [<release type>]
```
Comment (JoeyDeol) [inline packages/@storefront/core/CHANGELOG.md]: Need to add a release type.

Comment (jmbtutor) [inline packages/@storefront/core/CHANGELOG.md]: Add a proper release type.

Comment (jrmykolyn) [inline packages/@storefront/core/CHANGELOG.md]: Real release type tho?

File: packages/@storefront/core/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
```
Comment (jrmykolyn) [inline packages/@storefront/core/CHANGELOG.md]: That release type is a real release type!

**Pull request merged**